src/Compilers/Server/VBCSCompiler/Connection.cs

```diff
@@ -105,7 +105,7 @@ public async Task<ConnectionData> HandleConnection(bool allowCompilationRequests
                     return new ConnectionData(CompletionReason.CompilationNotStarted);
                 }
                 
-                if(request.ProtocolVersion != BuildProtocolConstants.ProtocolVersion)
+                if (request.ProtocolVersion != BuildProtocolConstants.ProtocolVersion)
                 {
                     return await HandleMismatchedVersionRequest(cancellationToken).ConfigureAwait(false);
                 }
```

Comment:
Also the line below.

Reply:
I went ahead and fixed that (directly through github UI)